iib/workers/tasks/opm_operations.py

```diff
@@ -193,6 +193,130 @@ def _serve_cmd_at_port(serve_cmd, cwd, port, max_tries, wait_time):
     raise IIBError(f'Index registry has not been initialized after {max_tries} tries')
 
 
+def _get_temp_index_db_file(base_dir, from_index):
+    """
+    Get path to temp index.db used for opm registry commands.
+
+    If index.db does not exist it will be created, ether by copying from from_index
+    or as creating empty index.db file if from_index is not set.
+
+    :param str base_dir: base directory where index.db file will be located.
+    :param str from_index: index image, from which we should copy index.
+    :return: Returns path to index.db located in base_dir.
+    :rtype: str
+    """
+    index_db_file = os.path.join(base_dir, get_worker_config()['temp_index_db_path'])
+
+    if os.path.exists(index_db_file):
+        log.debug('Temp index.db already exist for %s', from_index)
+        return index_db_file
+
+    log.info('Temp index.db does not exist yet for %s', from_index)
+    if from_index:
+        log.info('Using the existing database from %s', from_index)
+        return get_hidden_index_database(from_index, base_dir)
+    else:
+        log.info('Creating empty database file %s', index_db_file)
+        index_db_dir = os.path.dirname(index_db_file)
+        if not os.path.exists(index_db_dir):
+            os.makedirs(index_db_dir, exist_ok=True)
+        with open(index_db_file, 'w'):
+            pass
+
+    return index_db_file
+
+
+def opm_registry_deprecatetruncate(
+    base_dir,
+    index_db,
+    bundles,
+    from_index=None,
+    overwrite_target_index_token=None,
+    container_tool=None,
+):
+    """
+    Migrate SQLite database to File-Based catalog using opm command.
```

Comment:
in the name of the function is nothing mention about migration to FBC

Reply:
Thanks